spec/features/user_signs_up_spec.rb

```diff
@@ -18,11 +18,15 @@
     fill_in 'Email', with: 'john@example.com'
     fill_in 'Password', with: 'password'
     fill_in 'Password confirmation', with: 'password'
+    attach_file "Profile Photo", "#{Rails.root}/spec/support/images/images.jpeg"
 
     click_button 'Sign up'
 
+    user = User.last
+
     expect(page).to have_content('Welcome! You have signed up successfully.')
     expect(page).to have_content('Sign Out')
+    expect(user.profile_photo.file.filename).to eq("images.jpeg")
```

Comment:
Always think of a Capybara test in terms of what the user sees. If we upload a photo, then we should expect to see that photo in the page.